exercises/concept/lucas-numbers/.docs/instructions.md

```diff
@@ -0,0 +1,64 @@
+# Instructions
+
+You are a huge fan of the [Numberphile Youtube channel](https://www.youtube.com/watch?v=PeUbRXnbmms) and you just saw a cool video about the _Lucas Number Sequence_. You want to create this sequence using Elixir.
+
+While designing your function, you want to make use of _lazy evaluation_, so that you can generate as many numbers as you want, but only if you need to -- So you decide to use an enumerator:
+
+## 1. Create an enumerator
+
+When creating the `LucasNumbers` object, create an enumerator which you can access with `sequence`
+
+```ruby
+LucasNumbers.new.sequence
+# => #<Enumerator: #<Enumerator::Generator:0x000055c6d9793bc8>:each>
+```
+
+## 2. Return the first two numbers
+
+You know from the video that the first two numbers of the Lucas Numbers sequence are `2` then `1`. When the sequence enumerator's `next` method is called, return those first
+
+```ruby
+sequence = LucasNumbers.new.sequence
+sequence.next
+# => 2
+sequence.next
+# => 1
+```
+
+## 3. Create the generalized case
+
+You also know that after the first two numbers, you can generate the next number by adding the previous two numbers.
+
+```ruby
+sequence = LucasNumbers.new.sequence
+sequence.next
+# => 2
+sequence.next
+# => 1
+sequence.next
+# => 3
+sequence.next
+# => 4
+```
+
+## 4. Return a finite sequence
+
+To avoid repetitive calls to `sequence.next`, you decide to implement a `to_a` method which takes an integer and returns an array with the first `n` numbers from the Lucas Numbers sequence
```

Comment:
I know this is draft, but wanted to bump this, but not just bump it, give something useful while doing that.
  
This sentence does not seem to end here.  If it is not meant to end, and the code snippet that follows is part of that, perhaps a semi-colon or a colon is called for?

But also, this suggestion because otherwise we are encouraging a variable name instead of indicating the common use of _n_ to indicate any number of things:

```suggestion
To avoid repetitive calls to `sequence.next`, you decide to implement a `to_a` method which takes an integer and returns an array with the first _n_ numbers from the Lucas Numbers sequence
```

Perhaps emboldened.  Not sure.